packages/babel-plugin/src/css-prop/__tests__/index.test.tsx

```diff
@@ -639,12 +681,44 @@ describe('css prop', () => {
       expect(actual).toInclude('.cc-hash-test{font-size:20px}');
     });
 
+    it('should inline constant object property value', () => {
+      const actual = transform(`
+        import '@compiled/core';
+        import React from 'react';
+
+        const colors = { error: 'red' };
+
+        <div css={{ color: colors.error }}>hello world</div>
+      `);
+
+      expect(actual).toInclude('.cc-hash-test{color:red}');
+    });
+
+    it('should inline nested constant object property value', () => {
+      const actual = transform(`
+        import '@compiled/core';
+        import React from 'react';
+
+        const theme = {
+          colors: {
+            light: {
+              primary: '#fff',
+            }
+          }
+        };
+
+        <div css={{ color: theme.colors.light.primary }}>hello world</div>
+      `);
+
+      expect(actual).toInclude('.cc-hash-test{color:#fff}');
+    });
+
     it('should persist suffix of dynamic property value from objects into inline styles', () => {
       const actual = transform(`
         import '@compiled/core';
         import React from 'react';
 
-        const heading = {
+        let heading = {
```

Comment:
Are we checking something with `let` instead of `const`? 🤔

Reply:
if its const we will inline - if its let (or var) we won't

so i changed this so the test was still the same :P